docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3.8'
+
+services:
+  auth:
+    image: node:latest
```

Comment:
May be better to use a specific version?

Reply:
i don't have strong opinion, GPT helped to create this file